src/main/java/org/wildfly/security/auth/client/ElytronXmlParser.java

```diff
@@ -1259,7 +1259,7 @@ static void parseKeyStoreType(ConfigurationXMLStreamReader reader, final Version
                         resourceSource = parseNameType(reader);
                         break;
                     }
-                    case "uri": {
+                    case "load-from": {
```

Comment:
For the purposes of backwards compatibility the implementation in the parser tends to win over the the schema definition.

A configuration written correctly against the schema would not work so we know that know users can possibly be using it, however if they are aware of the discrepency they can create a config based on the parser implementation.

Versions 1.0 and 1.0.1 of the schema should be updated to reflect reality, i.e. the schema should be updated to the element is called 'uri'.

We could then choose to correct this in version 1.1 of the schema so uri would be supported if (!xmlVersion.isAtLeast(VERSION_1_1)) whilst load-from would then be supported if (xmlVersion.isAtLeast(VERSION_1_1)) alternatively we could support both forms in 1.1 and add some documentation in the schema to say the uri form is deprecated.